README.md

```diff
@@ -411,6 +411,14 @@ the workloads define the following in their cloudinit documents.
 ```
 package_upgrade: {{with .PackageUpgrade}}{{.}}{{else}}false{{end}}
 ```
+The -debug option can be used to follow qemu boot logs by connecting to port
```

Comment:
Could you choose a different name for this option.  The reason is that debug is already defined for the create command and it has a different purpose, to dump out the contents of the cloud-init file.  Now it's overloaded.  What about something like debug-port and have the user pass in the port number?  That way the user can chose a different port if 9999 is already being used.

Reply:
You would need to add this new flag to createFlags as well.

Reply:
Definitely.